pkg/client/userd/trafficmgr/session.go

```diff
@@ -1037,9 +1034,10 @@ func (s *session) connectRootDaemon(ctx context.Context, nc *rootdRpc.NetworkCon
 		}()
 		rd = rootdRpc.NewDaemonClient(conn)
 
+		tmTimeout := client.GetConfig(ctx).Timeouts().Get(client.TimeoutTrafficManagerConnect)
 		for attempt := 1; ; attempt++ {
 			var rootStatus *rootdRpc.DaemonStatus
-			tCtx, tCancel := context.WithTimeout(ctx, 15*time.Second)
+			tCtx, tCancel := context.WithTimeout(ctx, tmTimeout/2)
```

Comment:
I assume we divide by 2 to effectively keep the previous value (assuming the default 60s was not overriden).

Reply:
No, it actually changes value from 15s to 30s :-) (and makes it dependent on the `trafficManagerConnect`).

Reply:
ah yeah, I meant 60 (the default value as I mentioned there) divided by 2 which is indeed 30. I don't know why but I saw `15*time.Second` as `2*15*time.Second` in my head